### PR TITLE
Add about me, projects, links, and cat image gallery

### PR DESCRIPTION
**Pull request by kinsaurralde**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -5,37 +5,80 @@
...
-      <h2>Links</h2>
-      <p>This section will have links to other profiles</p>
-    </div>
+    <h1 class="centered">My Portfolio</h1>
     <div class="subsection">
       <h2>About Me</h2>
-      <p>This is my portfolio.</p>
-      <p>My name is Kyle Insaurralde</p>
+      <p>My name is Kyle Insaurralde. I am a junior at UC Davis. My favorite programming language is python. For
+      fun, I like to play minecraft.</p>
```
Comment (huzcn) [inline portfolio/src/main/webapp/index.html]: You can use 4 space indention in wrapped lines. See here: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -5,37 +5,80 @@
...
     </div>
     <div class="subsection">
       <h2>Projects</h2>
-      <p>This section will have my projects</p>
+      <div class="left-space">
+        <h3>Website</h3>
+        <div class="left-space">
+          <p>I have another project website that I add stuff to. It contains the projects listed below along
+          with a few others.</p>
```
Comment (huzcn) [inline portfolio/src/main/webapp/index.html]: Same for this line and other lines in this file

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+    ['Hi', 'Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!', 'Greeting', "According to all known laws of aviation, there is no way a bee should be able to fly. Its wings are too small to get its fat little body off the ground. The bee, of course, flies anyway because bees don't care what humans think is impossible."];
```
Comment (huzcn) [inline portfolio/src/main/webapp/script.js:24]: You can keep the 4 space indention here

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -17,7 +17,7 @@
...
```
Comment (huzcn) [inline portfolio/src/main/webapp/script.js]: If the line is too long, you can optionally wrap into multiple lines to improve readability.

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -26,3 +26,86 @@ function addRandomGreeting() {
...
+const MAX_HEIGHT = 600;
+
+changeCatPhoto(0);
+
+/**
+ * Switches image in cat gallery
+ * @param {number} num
+ */
+function changeCatPhoto(num) {
+  if (catPhotos[num]["img"] == null) { // Photo only needs to be loaded once
```
Comment (huzcn) [inline portfolio/src/main/webapp/script.js]: For the comment style, I'm leaning to use /* */ which is indicated (but not explicitly required) in the style guide: https://google.github.io/styleguide/htmlcssguide.html#Section_Comments

**Review by huzcn — CHANGES_REQUESTED** [review]
Nice! Added some comments mostly about styles.

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -16,8 +16,11 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const greetings = [
+    'Hi', 'Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!',
+    'Greeting',
+    'According to all known laws of aviation, there is no way a bee should be able to fly. Its wings are too small to get its fat little body off the ground. The bee, of course, flies anyway because bees don\'t care what humans think is impossible.'
```
Comment (huzcn) [inline portfolio/src/main/webapp/script.js:22]: Is this auto-formatted by the tool I recommended? I think the wrapped lines of this "greetings" variable should be 4 spaces indention, if you notice this in the original code. Feel free to correct me if I'm wrong

**Review by huzcn — COMMENTED** [review]
Nice!

File: portfolio/src/main/webapp/script.js (hunk 6 above)
Comment (kinsaurralde) [reply portfolio/src/main/webapp/script.js:22]: This is what the auto-formatter did

Comment (huzcn) [reply portfolio/src/main/webapp/script.js:22]: Looks like against style used internally, but there is no need to too picky on this, as long as the style is consistent across the whole codebase.

**Review by huzcn — APPROVED** [review]

**Pull request merged**